[31.0.0-beta2] - source tarball is missing cmake/common/buildnumber.cmake

### Operating System Info

Other

### Other OS

Exherbo Linux

### OBS Studio Version

31.0.0-beta1

### OBS Studio Version (Other)

31.0.0-beta2

### OBS Studio Log URL

-

### OBS Studio Crash Log URL

_No response_

### Expected Behavior

Being able to run cmake without hitting any error.

### Current Behavior

```
CMake Error at cmake/common/bootstrap.cmake:61 (include):
  include could not find requested file:

    buildnumber
Call Stack (most recent call first):
  CMakeLists.txt:3 (include)
```

The source tarball https://github.com/obsproject/obs-studio/releases/download/31.0.0-beta2/OBS-Studio-31.0.0-beta2-Sources.tar.gz is missing the file(s)  https://github.com/obsproject/obs-studio/blob/31.0.0-beta2/cmake/common/buildnumber.cmake (and https://github.com/obsproject/obs-studio/blob/31.0.0-beta2/cmake/common/buildspec_common.cmake).

### Steps to Reproduce

1. cmake ../obs-studio-31.0.0-beta2-sources

### Anything else we should know?

_No response_

--- a/cmake/linux/cpackconfig.cmake
+++ b/cmake/linux/cpackconfig.cmake
@@ -16,7 +16,7 @@ endif()
 
 set(CPACK_SOURCE_PACKAGE_FILE_NAME "obs-studio-${CPACK_PACKAGE_VERSION}-sources")
 set(CPACK_SOURCE_GENERATOR "TGZ")
-set(CPACK_SOURCE_IGNORE_FILES "/.git" "/.*build.*" "/.ccache" "/.deps")
+set(CPACK_SOURCE_IGNORE_FILES "/.git" "/build_${CMAKE_SYSTEM_PROCESSOR}" "/.ccache" "/.deps")
 set(CPACK_ARCHIVE_THREADS 0)
 
 if(OS_LINUX)

